storage/range.go

```diff
@@ -1709,6 +1604,97 @@ func (r *Range) SetHardState(st raftpb.HardState) error {
 		proto.ZeroTimestamp, nil, &st)
 }
 
+// AdminSplit divides the range into into two ranges, using either
```

Comment:
Just a move because the Raft stuff got put right in between the split and merge methods.